medcat/cat.py

```diff
@@ -1056,7 +1087,7 @@ def multiprocessing_pipe(self,
             import torch
             torch.set_num_threads(1)
 
-        in_data = list(in_data) if isinstance(in_data, types.GeneratorType) else in_data
+        in_data = list(in_data) if isinstance(in_data, Iterable) else in_data
         n_process = nproc if nproc is not None else min(max(cpu_count() - 1, 1), math.ceil(len(in_data) / batch_factor))
         batch_size = batch_size if batch_size is not None else math.ceil(len(in_data) / (batch_factor * abs(n_process)))
 
```

Comment:
why skip mp_cons? is it not necessary for private functions?

Reply:
Done.